lib/AST/ASTContext.cpp

```diff
@@ -4340,6 +4340,107 @@ CanGenericSignature ASTContext::getExistentialSignature(CanType existential,
   return genericSig;
 }
 
+GenericSignature *ASTContext::getOverrideGenericSignature(ValueDecl *base,
+                                                          ValueDecl *derived) {
```

Comment:
It might be better if this was two methods, with the second one taking the two generic signatures and a Type.